### ISSUE TITLE
Implement Picture Articles

### ISSUE THREAD

**Issue by JamieB-gu** [guardian/dotcom-rendering#5013]
We'll want to implement picture articles using either dotcom (frontend) as a reference design, or based on a new design created by the Design team. It might be possible to re-use the `standard.tsx` layout file for this, or it might be necessary to create a new one.

Details on the features of picture articles, and some examples, can be found in the milestone description. This work is dependent on #4941.

**Comment by georgeblahblah** [comment]
Won't do now that we are rendering apps articles in DCR
